### DISCUSSION TITLE
658

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#895] [Comments]
# 658

팀에 공유한 내용인데 블로그에도 정리한다. 오늘 주변의 추천으로 희종님의 글을 봤다. https://ahnheejong.name/articles/i-was-wrong-test/ 예전에 어느책에서 봤던 내용인데 (제목은 까먹었다 ㅠㅠ) 나 스스로 "내가 틀렸네요" 라고 입밖으로 낼 수 있는 용기가 있다면 그사람은 성장하는중이다 라는 이야기를 본적이 있다. 그래서 그 이후로 "제가 잘못알고 있었네요" 라고 이야기를 해야하는 순간이 오면 진짜 민망하고, 인정하기 싫었지만, "그래도 이걸 이야기하지 않으면 나는 성장하지 못하겠지? 그게 더 무서운거 아닌가?" 라는 생각으로 어떻게든 내뱉었다. 어떤 일을 하는데 있어 개인이 성장하려면 내가 틀렸음을 인정하고, 이걸 입밖으로 낼 용기를 가지는 것이 필요하다. 이걸 ..

https://jojoldu.tistory.com/658

**Comment by 62hoon99** [comment]
좋은 글 감사합니다.
